test/clojure_tictactoe/game/game_play_test.clj

```diff
@@ -0,0 +1,17 @@
+(ns clojure-tictactoe.game.game-play-test
+  (:require [clojure.test :refer :all]
+            [clojure.string :as string]
+            [clojure-tictactoe.helpers :as helper]
+            [clojure-tictactoe.game.game-play :refer :all]))
+(def empty-board
+  [0 1 2 3 4 5 6 7 8])
+
+(def board-x-at-8
```

Comment:
I'd suggest avoiding using `def` assignments when you're only using them once or twice.

Reply:
✅